### PR TITLE
5812 - Replace Edit Signature with Remove Signature

### PR DESCRIPTION
**Pull request by matthopson**
Screen Shot 2020-07-21 at 2 13 15 PM

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/computeds/draftDocumentViewerHelper.js [hunk 1]
```diff
@@ -81,7 +81,8 @@ export const draftDocumentViewerHelper = (get, applicationContext) => {
     showDocumentNotSignedAlert,
     showEditButtonNotSigned: showEditButtonForRole && !documentIsSigned,
     showEditButtonSigned: showEditButtonForRole && documentIsSigned,
-    showEditSignatureButton:
-      showApplyEditSignatureButtonForRole && showEditSignatureButtonForDocument,
+    showRemoveSignatureButton:
+      showApplyEditSignatureButtonForRole &&
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/draftDocumentViewerHelper.js]: Could you change this name also to `showApplyRemoveSignatureButtonForRole`

File: web-client/src/presenter/computeds/messageDocumentHelper.js [hunk 2]
```diff
@@ -111,9 +111,10 @@ export const messageDocumentHelper = (get, applicationContext) => {
       showEditButtonForRole && showEditButtonForDocument && documentIsSigned,
     showEditCorrespondenceButton:
       showEditButtonForRole && showEditButtonForCorrespondenceDocument,
-    showEditSignatureButton:
-      showApplyEditSignatureButtonForRole && showEditSignatureButtonForDocument,
     showNotServed,
+    showRemoveSignatureButton:
+      showApplyEditSignatureButtonForRole &&
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/messageDocumentHelper.js]: Same with this one - `showApplyRemoveSignatureButtonForRole`

File: wikiwiki/Drafts.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ on click of "Save" on Edit Order screen, app navigates to Apply Signature screen
 * On click of "Skip Signature", app navigates back to Draft Document View with success message displayed:"[Order/Notice title] updated."
 * On click of "Save Signature", app navigates back to Draft Document View with success message displayed: "[Order/Notice title] updated."
 
-**Edit Signature**
+**Remove Signature**
 * Action displays. if the document has been signed
 * Displays the Editing Signature modal, on Yes, Continue- navigates to Apply Signature screen
 * when user completes action (Saves, or Skips signature), navigate back to Draft Document View with success message displayed: "[Order/Notice title] updated."
```
Comment (rachaelparris) [inline wikiwiki/Drafts.md:34]: Looks like this might need some more updating (maybe by Kristen?)

**Review by rachaelparris — APPROVED** [review]

File: wikiwiki/Drafts.md (hunk 3 above)
Comment (matthopson) [reply wikiwiki/Drafts.md:34]: Oh, yes I vote @klohman

Comment (klohman) [reply wikiwiki/Drafts.md:34]: I usually try to update the wiki after UX review? I have some other updates to make and can add this in with those (if that works)

Comment (matthopson) [reply wikiwiki/Drafts.md:34]: Yes, please, and thank you.

**Pull request merged**